### ISSUE TITLE
False negative for the DL3013 rule

### ISSUE THREAD

**Issue by tymonx** [zabio3/godolint#17]
This simple example fails:

```Dockerfile
ARG YAML_LINT_VERSION=v1.26.3

RUN pip install --no-cache-dir yamllint=="${YAML_LINT_VERSION:-v1.26.3}"
```

Error:

```plaintext
DL3013 Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`. 
```
